Fix documentation for Secret Stores query parameters. (#3593)
dapr/docs: daprdocs/content/en/reference/components-reference/supported-secret-stores/kubernetes-secret-store.md

## Spec metadata fields
For the Kubernetes secret store component, there are no metadata attributes.

## Optional per-request metadata properties

The following [optional query parameters]({{< ref "secrets_api#query-parameters" >}}) can be provided to Kubernetes secret store component:

Query Parameter | Description
--------- | -----------

> `metadata.namespace`| The namespace of the secret. If not specified, the namespace of the pod will be used.

Reviewer: ```suggestion
`metadata.namespace`| The namespace of the secret. If not specified, the namespace of the pod is used.
```
